NewForce-Cohort-4/tabloidmvc-fluffy-shield
# JN - Add a Tag to a Post

# Description

> ​As an author I would like to be able to associate one or more Tags with one of my Posts so that readers can easily see the Post topics at a glance.

Updated the Post Details view with a sidebar featuring tags added to the post and providing a Tag Management button. Constructed a Select Tags page that displays all tags, user can select multiple tags and then save those to a post. Once selected tags are saved, the user is redirected to the Details page for the related post.
​
Fixes [#17 Add a Tag to a Post](https://trello.com/c/rijLZZyu/17-17-add-a-tag-to-a-post)
​
## Type of change
​
​
# How Has This Been Tested?
​
- `git` fetch all & checkout PR branch `jn-add-tag`
- If prompted, log in with user account: `user@example.com`
- From the NavBar, click `All Posts`
- On a displayed post, click `View Details` in the post options
- On the right side of the Post details, click `Tag Management`
- Choose multiple tags from the displayed option and click `Save`
- Confirm the redirect back to the related Post Details with the selected Tags displayed.
​
# Checklist:
​
